MekHQ/src/mekhq/campaign/personnel/Person.java

```diff
@@ -165,6 +166,9 @@
     public static final int DESIG_NUM     = 14;
     
     public static final String LOGTYPE_MEDICAL = "med";
+    
+    private static final HashMap<Integer, Integer> MECHWARRIOR_AERO_RANSOM_VALUES; 
+    private static final HashMap<Integer, Integer> OTHER_RANSOM_VALUES;
```

Comment:
Unless you're using features specific to HashMap (that aren't in Map), you should declare these as Map.
From _Effective Java 2nd Edition, Item 52:_

> If appropriate interface types exist, then parameters, return values, and fields should all be declared using interface types. If you get into the habit of using interface types, your program will be much more flexible. It is entirely appropriate to refer to an object by a class if no appropriate interface exists.